### ISSUE TITLE
Swipe to change song in now playing

### ISSUE THREAD

**Issue by Coolbeans1388** [z-huang/InnerTune#1017]
### Checklist

- [X] I've checked that there is no other issue about this feature request.
- [X] This issue contains only one feature request.
- [X] The title of this issue accurately describes the feature request.

### Feature description

Latest update removed ability to swipe to change song on now playing screen.

### Why do you want this feature?

I miss this feature from last version.

**Comment by z-huang** [comment]
Yeah it's removed because it's not stable. Once I find a better way to implement, I'll add it back.

**Comment by z-huang** [comment]
Yeah, it's related to the queue. When the queue is changed, the `HorizontalPager` will do the animation. Somehow it detects a move action by mistake and change the track accordingly.

**Comment by Vishcot** [comment]
what if we try to integrate Bottom to Top swiping gestures for changing songs like Resso App, Wynk and Gaana app have now implemented, it is much more convenient and more interactive @z-huang

**Comment by d4f5409d** [comment]
Retro Music Player has it nicely implemented, you may take a look at their code
